.gitignore

```diff
@@ -137,6 +137,7 @@ gradle-app.setting
 
 # Python
 venv
```

Comment:
maybe i'm wrong, it is still necessary?

Reply:
no, we'll keep .venv, so venv is unnecessary

Reply:
ok right, you can remove it